Apps/AdminWebClient/src/AdminWebClient.csproj

```diff
@@ -18,9 +18,9 @@
   </PropertyGroup>  
 
   <ItemGroup>
+    <PackageReference Include="CsvHelper" Version="15.0.5" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="3.1.6" />
     <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="3.1.6" />
-    <PackageReference Include="ServiceStack.Text" Version="5.9.2" />
```

Comment:
Just in case you have removed this by accident.

Reply:
I introduced and Tiago didn't use it.  Removed to keep code tidy.